plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ActiveMapTracker.java

```diff
@@ -288,4 +329,11 @@ public void earlyStartup() {
         }
     }
 
+    /**
+     * @return Collection of open MapParts.
+     */
+    public Collection<MapPart> getOpenMapParts() {
```

Comment:
@sschulz92 Thanks for this changeset! Wondering if you can also contribute test cases e.g. ActiveMapTrackerTest to verify behaviour